subscriber_notifee.go

```diff
@@ -56,7 +56,7 @@ func newSubscriberNotifiee(dht *IpfsDHT) (*subscriberNotifee, error) {
 	dht.plk.Lock()
 	defer dht.plk.Unlock()
 	for _, p := range dht.host.Network().Peers() {
-		protos, err := dht.peerstore.SupportsProtocols(p, dht.protocolStrs()...)
+		protos, err := dht.peerstore.SupportsProtocols(p, dht.clientProtocolStrs()...)
```

Comment:
Please add docs explaining why we check the peerstore for these protocols i.e. We will ONLY add peers who speak the "new" protocol to the Routing Table.